sysdig/resource_sysdig_secure_cloud_auth_account_test.go

```diff
@@ -0,0 +1,130 @@
+//go:build tf_acc_sysdig_secure || tf_acc_sysdig_common
+
+package sysdig_test
+
+import (
+	b64 "encoding/base64"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/draios/terraform-provider-sysdig/sysdig"
+)
+
+func TestAccSecureCloudAuthAccount(t *testing.T) {
+	rText := func() string { return acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum) }
+	accID := rText()
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			if v := os.Getenv("SYSDIG_SECURE_API_TOKEN"); v == "" {
+				t.Fatal("SYSDIG_SECURE_API_TOKEN must be set for acceptance tests")
+			}
+		},
+		ProviderFactories: map[string]func() (*schema.Provider, error){
+			"sysdig": func() (*schema.Provider, error) {
+				return sysdig.Provider(), nil
+			},
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: secureCloudAuthAccountWithID(accID),
+			},
+			{
+				Config: secureCloudAuthAccountMinimumConfiguration(accID),
+			},
+			{
+				ResourceName:      "sysdig_secure_cloud_auth_account.sample",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func secureCloudAuthAccountWithID(accountID string) string {
+	return fmt.Sprintf(`
+resource "sysdig_secure_cloud_auth_account" "sample" {
+  provider_id   = "sample-%s"
+  provider_type = "PROVIDER_GCP"
+  enabled       = "true"
+}
+`, accountID)
+}
+
+func secureCloudAuthAccountMinimumConfiguration(accountID string) string {
```

Comment:
What's the different between this one and secureCloudAuthAccountWithID?

Reply:
Yeah, it can be a single test. I'll push a fix.

Reply:
Done.